README.md

```diff
@@ -85,6 +85,27 @@ digraph foo {
 ....
 ```
 
+## Using Your Own Kroki
+
+By default this extension sends information and receives diagrams back from `https://kroki.io`.
+
+For significant usage or many users we recommend setting up your own server (and also if sending information to an external server is considered an information security risk).
+
+This is done using the `kroki-server-url` attribute.
+Typically this is at the top of the document (under the title):
+
+```adoc
+:kroki-server-url: http://my-server-url:port
+```
+
```

Comment:
Maybe we should add a note to say that it's also possible to define the attribute using the API:

```js
asciidoctor.convertFile('file.adoc', { attributes: { 'kroki-server-url': 'http://my-server-url:port' } })
```